HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -1516,14 +1522,21 @@ def self.add_ee_for_vent_fan_power(model, obj_name, sup_fans = [], exh_fans = []
       end
     end
 
+    # Availability Schedule
+    avail_sch = model.alwaysOnDiscreteSchedule
+    if not power_outage_periods.empty?
+      avail_sch = ScheduleRulesetConstant.new(model, obj_name + ' schedule', 1.0, Constants.ScheduleTypeLimitsFraction, off_periods: power_outage_periods)
+      avail_sch = avail_sch.schedule
+    end
```

Comment:
How about renaming `ScheduleRulesetConstant` to `ScheduleConstant` and have it either return:
- `model.alwaysOnDiscreteSchedule` if always 1
- `model.alwaysOffDiscreteSchedule` if always 0
- `OpenStudio::Model::ScheduleConstant` if always something other than 0 or 1
- `OpenStudio::Model::ScheduleRuleset` if not constant

Then this code becomes simplified.

Reply:
Good idea.